app/src/main/java/com/pilot51/voicenotify/ui/TextField.kt

```diff
@@ -0,0 +1,323 @@
+            modifier = Modifier.padding(0.dp)
+                .clickable(
+                    onClick = { onValueChange("") },
+                ),
+            contentDescription = "Clear text"
+        )
+    },
+    prefix: @Composable (() -> Unit)? = null,
+    suffix: @Composable (() -> Unit)? = null,
+    supportingText: @Composable (() -> Unit)? = null,
+    isError: Boolean = false,
+    visualTransformation: VisualTransformation = VisualTransformation.None,
+    keyboardOptions: KeyboardOptions = KeyboardOptions.Default,
+    keyboardActions: KeyboardActions = KeyboardActions.Default,
+    singleLine: Boolean = false,
+    maxLines: Int = if (singleLine) 1 else Int.MAX_VALUE,
+    minLines: Int = 1,
+    interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
+    shape: Shape = TextFieldDefaults.shape,
+    colors: TextFieldColors = TextFieldDefaults.colors(
+        focusedContainerColor = VoicenotifyTheme.colors.textFieldContainer,
+        unfocusedContainerColor = VoicenotifyTheme.colors.textFieldContainer,
+        disabledContainerColor = Color.Transparent,
+        focusedIndicatorColor = Color.Transparent,
+        disabledIndicatorColor = Color.Transparent,
+        unfocusedIndicatorColor = Color.Transparent
+    )
+) {
+    BasicTextField(
+        value = value,
+        onValueChange = onValueChange,
+        modifier = modifier.semantics {
+            if (contentDescription != null) {
+                this.contentDescription = contentDescription
+            }
+        },
+        enabled = enabled,
+        readOnly = readOnly,
+        textStyle = textStyle,
+        keyboardOptions = keyboardOptions,
+        keyboardActions = keyboardActions,
+        visualTransformation = visualTransformation,
+        interactionSource = interactionSource,
+        singleLine = singleLine,
+        decorationBox = @Composable { innerTextField ->
+            TextFieldDefaults.DecorationBox(
+                value = value,
+                visualTransformation = visualTransformation,
+                innerTextField = innerTextField,
+                placeholder = placeholder,
+                label = label,
+                leadingIcon = leadingIcon,
+                trailingIcon = {
+                    if (value.isNotEmpty()) {
+                        if (clearIcon != null) {
+                            clearIcon()
+                        }
+                    }
+                    trailingIcon?.invoke()
+                },
+                prefix = prefix,
+                suffix = suffix,
+                supportingText = supportingText,
+                singleLine = singleLine,
+                enabled = enabled,
+                isError = isError,
+                interactionSource = interactionSource,
+                colors = colors,
+                contentPadding = PaddingValues(0.dp, 0.dp, 0.dp, 0.dp),
+            )
+        }
+    )
+//    BasicTextField(
```

Comment:
Please avoid leaving commented code.
There is commented code in several places and I won't point out all of them, so please go over your changes and remove or uncomment them.